libs/entry-components/styles/partials/core/components/search-filter/_general.scss

```diff
@@ -1,9 +1,6 @@
 @use 'scss-foundation/src/modules/display/items';
 
 .search-form-container {
+	padding-bottom: 10px;
 	@include items.align-center();
```

Comment:
Place @include directive at the beginning of a code block, before any other declarations for better readability and consistency